public/index.html

```diff
@@ -12,6 +12,6 @@
 </head>
 <body>
   <noscript>In order to view this documentation page, you have to enable JavaScript in your web browser.</noscript>
-  <div id="app"></div>
+  <div id="app" data-root="DATAROOT"></div>
```

Comment:
See the comment above about replacing the whole tag instead of just the attribute (which doesn't have to be mandatory).